### PR TITLE
Replaced devbots with issue-labeler

### PR DESCRIPTION
**Pull request by jpkrohling**
The bot that used to add the 'needs-triage' label to new issues stopped working and I can't figure out why. Looks like GitHub has a more official action that does something similar, so, this PR is replacing the 'devbots' with the issue-labeler.

See https://github.com/jaegertracing/jaeger-operator/pull/1251

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Review by objectiser — APPROVED** [review]

**Comment by yurishkuro** [comment]
fwiw I don't find `needs-triage` label particularly useful, we don't seem to pay much attention to it

**Comment by jpkrohling** [comment]
I use it quite a lot to know which issues were reported and that nobody looked at. For the operator, I review them once a day and remove this label, adding other ones, but I noticed that some issues in the main repository are closed while still with `needs-triage`. I need to do a better job in this repository here :-)

**Comment by yurishkuro** [comment]
I used to remove this label when responding, but found it to be too much hustle. I think the semantics of this label is not very well defined either. If we're going after the typical customer support model, then really the labels need to keep changing from "waiting for support" to "waiting for OP feedback". If needs-triage is simply "has anyone looked at it", then perhaps the bot should remove the label automatically once any of the maintainers reply to the ticket.

**Comment by jpkrohling** [comment]
> then perhaps the bot should remove the label automatically once any of the maintainers reply to the ticket.

I think it's actually doable.

**Pull request merged**
